javascript/browser-apis/reading-writing/blobs.md

```diff
@@ -33,34 +20,44 @@ links:
 ---
 ## Content
 
-A *blob* object represents an "almost file" object of raw data. It represents data that isn't in JavaScript's standard format. You use *FileReader* to read data from a blob.
+A binary large object, or *blob*, represents a file-like object of immutable raw data. It represents data that isn't in JavaScript's standard format. You use `FileReader` to read data from a blob.
 
 To create a blob, use the blob constructor:
 ```javascript
-var debug = {hello: "world"};
-var blob = new Blob([
-  JSON.stringify(debug, null, 2)
-], {type : 'application/json'});
+let debug = {
+  hello: "world",
+};
+
+let blob = new Blob(
+  [JSON.stringify(debug, null, 2)],
+  {
+    type: "application/json",
+  }
+);
 ```
 
-To create a subset of a blob, use the *slice()* method:
-```javascript
-var blob = instanceOfBlob.slice([
-  start [, end [, contentType]]
-]);
-```
+A blob can be very large, so this means it can contain audio and video data. They can be created dynamically, and used as files when coupled with blobl URLs.
```

Comment:
this confused me a bit. what are blob URLs ? ( btw there's a "blobl" tpyo)
also what's meant by "created dynamically"?